src/equations/kdv_1d.jl

```diff
@@ -181,3 +181,72 @@ function rhs!(dq, q, t, mesh, equations::KdVEquation1D, initial_condition,
 
     return nothing
 end
+
+function rhs_split_1!(dq, q, t, mesh, equations::KdVEquation1D, initial_condition,
```

Comment:
```suggestion
function rhs_split_stiff!(dq, q, t, mesh, equations::KdVEquation1D, initial_condition,
```
I think that's more descriptive.